wagtail/wagtailadmin/static_src/wagtailadmin/js/blocks/sequence.js

```diff
@@ -79,6 +79,10 @@ CODE FOR SETTING UP SPECIFIC UI WIDGETS, SUCH AS DELETE BUTTONS OR MENUS, DOES N
             return parseInt(countField.val(), 10);
         };
 
+        self.getRealCount = function() {
```

Comment:
Let's come up with a better name for this. The existence of a function called `getRealCount` just tells me "there's something subtly wrong with the result of `getCount`, but we're not going to tell you what it is... you have to read the implementation to find out".

If you think that `getCount` is badly named (and it probably is!) then we should fix it, not just build on top of that mistake. Perhaps `getCount` should become `getCountIncludingDeleted` or `getCurrentCountIndex`, so that the new function can be `getCount`?

Reply:
I fully agree, I just didn't want to break anything by changing getCount, but if you're find with me doing a commit to rename the function first (I'll rebase this work onto that), that works better. Just need to settle on a name.

Reply:
Another option... which I'm less inclined towards is to expose members itself as self._members

I can then get the length from there, but a function for count is much more useful here, and elsewhere.